src/rc.ts

```diff
@@ -240,9 +240,7 @@ export class RC {
   }
 
   setDataconnect(localConnectionString: string) {
-    if (!this.data.dataconnectEmulatorConfig) {
```

Comment:
@joehan confirming this doesn't break anything in cli

Reply:
Seems like this was previously broken, LGTM